Tasks/Npm/npmtask.ts

```diff
@@ -4,89 +4,113 @@ import path = require('path');
 import url = require('url');
 import tl = require('vsts-task-lib/task');
 import trm = require('vsts-task-lib/toolrunner');
+import nutil = require('nuget-task-common/utility');
+
 var extend = require('util')._extend;
 
 interface EnvironmentDictionary { [key: string]: string; }
 
-tl.setResourcePath(path.join( __dirname, 'task.json'));
+tl.setResourcePath(path.join(__dirname, 'task.json'));
 
 executeTask();
 
 async function executeTask() {
 
-    var cwd = tl.getPathInput('cwd', true, false);
-    tl.mkdirP(cwd);
-    tl.cd(cwd);
-
-    var command = tl.getInput('command', true);
-    if (command.indexOf(' ') >= 0) {
-        tl.setResult(tl.TaskResult.Failed, tl.loc('InvalidCommand'));
-        return;
+    try {
+        var filePath = tl.getPathInput('cwd', true, false);
+        var dirList: string[] = [];
+        if (filePath.indexOf("*") !== -1 || filePath.indexOf("?") !== -1) {
```

Comment:
I guess resolveFilterSpec should also handle the case where 2 paths are separated by ;
can you please verify this.